py3status/modules/whatismyip.py

```diff
@@ -58,17 +66,23 @@ def on_click(self, event):
     def _get_my_ip(self):
         """
         """
+        country = ""
         try:
+            if '{country}' in self.format:
```

Comment:
please use `if self.py3.format_contains(self.format, 'country'):` that way we don't have problems with odd formats like `{country:10}`